### ISSUE TITLE
Brainstorm

### ISSUE THREAD

**Issue by wilsonpinto** [RefugeesWork/mobile-app#2]
Leave here your ideas. 

Let's start simple for first version and then implement more features.

**Comment by serapath** [comment]
Maybe the first think would be to just have a "wall of jobs" - those which are listed on refugeeswork.com
I think Nina can open an API for it :-)

I would probably just use a module like `minixhr` (npm install minixhr) and query the API to get all the posted jobs and show them. Then throw some styling in to make it look nice and then use cordova to convert it to an android app.
But probably ionic as a lot of cool stuff out of the box, so then maybe thats great to use :-)

**Comment by wilsonpinto** [comment]
I agree, later i will put some list of endpoints that i gonna need for start. @ninabreznik some json return from ruby :)

@serapath if we use ionic it already take care of it for us :) But yeah that is great stuff.

**Comment by serapath** [comment]
sure - great :-) i'm curious

**Comment by serapath** [comment]
any news? :-)
